File: Level2/financial record.cpp

#include <iostream>
#include <string>

using namespace std;

struct Member {
    string name;
    string id;
    double totalSavings;
};

int main() {
    Member m;// object instantiation
    double currentInput;
    m.totalSavings = 0; // Initialize to zero

    // 1. Input Member Details
    cout << "Enter Member Name: ";
    getline(cin, m.name);
    
    cout << "Enter Member ID: ";
    cin >> m.id;

    // 2. Input Contributions for 6 Months
    cout << "\nEnter monthly contributions for the last 6 months:\n";
    for (int i = 1; i <= 6; i++) {
        cout << "Month " << i << ": ";
        cin >> currentInput;
        
        // Add current input to the running total
        m.totalSavings += currentInput;
    }

    // 3. Display Results
    cout << "\n--- Financial Summary ---" << endl;
    cout << "Member: " << m.name << " (" << m.id << ")" << endl;
    cout << "Total Savings: $" << m.totalSavings << endl;

    return 0;
}

File: Level2/financialrecord.py
# 1. Get Member Info
name = input("Enter Member Name: ")
member_id = input("Enter Member ID: ")
total_savings = 0.0

#Get 6 months of contributions
print("\nEnter contributions for 6 months:")
for i in range(1, 7):
    # Get input, convert to a number, and add to the total
    amount = float(input(f"Month {i}: "))
    total_savings += amount

# Result
print("\n--- Summary ---")
print("Member Name:", name)
print("Member ID:", member_id)
print("Total Savings: $", total_savings)